src/components/progress/themes/linear/shared/linear.progress.bootstrap.scss

```diff
@@ -10,56 +11,21 @@ $stripe-size: rem(5px);
     --track-size: #{rem(16px)};
 }
 
-[part~='striped'][part~='primary']:not([part~='indeterminate']) {
-    @include stripe-colors(var-get($theme, 'fill-color-default'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
-}
-
-[part~='striped'][part~='primary'] {
-    &::after {
-        @include stripe-colors(var-get($theme, 'fill-color-default'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
-    }
-}
-
-[part~='danger']:not([part~='indeterminate']) {
-    background-color: var-get($theme, 'fill-color-default');
-}
-
-[part~='striped'][part~='danger']:not([part~='indeterminate']) {
-    @include stripe-colors(var-get($theme, 'fill-color-danger'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
-}
-
-[part~='striped'][part~='danger'] {
-    &::after {
-        @include stripe-colors(var-get($theme, 'fill-color-danger'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
-    }
-}
-
-[part~='striped'][part~='warning']:not([part~='indeterminate']) {
-    @include stripe-colors(var-get($theme, 'fill-color-warning'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
-}
-
-[part~='striped'][part~='warning'] {
-    &::after {
-        @include stripe-colors(var-get($theme, 'fill-color-warning'), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
+// Define a mixin for generating striped styles
+@mixin striped-styles($part, $color-key) {
+    [part~='striped'][part~='#{$part}']:not([part~='indeterminate']) {
+        @include stripe-colors(var-get($theme, $color-key), var-get($theme, 'stripes-color'), $stripe-size, 45deg);
```

Comment:
Similar to my previous comment, I would appreciate if the background color was made a CSS variable. Also, this ignores the indeterminate state now, effectively changing the behavior. We don't want to have stripes in the indeterminate state.